src/main/groovy/au/org/ala/merit/config/ProgramConfig.groovy

```diff
@@ -96,6 +96,10 @@ class ProgramConfig implements Map {
         emailTemplate
     }
 
+    SettingPageType getDeclarationTemplate(declarationPageType) {
```

Comment:
The parameter "declarationPageType" is unnecessary here as it is managed by this class.